exchanges/btse/btse_wrapper.go

```diff
@@ -251,8 +251,18 @@ func (b *BTSE) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.P
 		if !m[x].Active {
 			continue
 		}
+
 		var pair currency.Pair
-		pair, err = currency.NewPairFromString(m[x].Symbol)
+		quote := m[x].Quote
+		if a == asset.Futures {
+			symSplit := strings.Split(m[x].Symbol, m[x].Base)
+			if len(symSplit) == 1 {
```

Comment:
I thought strings.Split always returned at least 1, but in the crazy event that both Symbol and Base are`""` it can return 0, so I will do
```go
	t.Log(len(strings.Split("", ""))) -> 0
```